client/src/Styles/Game.css

```
@@ -1,162 +1,167 @@
 .game-container {
-    width: 100%;
-    height: 100vh;
-    color: white;
+  width: 100%;
+  height: 100vh;
+  color: white;
 }
 
 .match-container {
-    height: 70vh;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 0 3%;
+  height: 70vh;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 3%;
 }
 
 .user-container {
-    display: flex;
-    flex-direction: column;
-    justify-content: space-between;
-    align-items: center;
-    width: 30%;
-    height: 40%;
-    font-size: 42px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  width: 30%;
+  height: 40%;
+  font-size: 42px;
 }
 
 .user-selection {
-    display: flex;
-    justify-content: space-between;
-    width: 100%;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
 }
 
 .user-selection div {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    background: none;
-    outline: none;
-    border: 0.2px solid white;
-    color: white;
-    width: 30px;
-    height: 15px;
-    padding: 20px 60px;
-    cursor: pointer;
-    font-size: 21px;
-}
-
-.selection-rock, .selection-paper, .selection-scissors {
-    position: relative;
-    overflow: hidden;
-}
-
-.selection-rock:hover::before, .selection-paper:hover::before, .selection-scissors:hover::before {
-    width: 100%;
-}
-
-.selection-rock::before, .selection-paper::before, .selection-scissors::before {
-    position: absolute;
-    top: 0px;
-    left: 0px;
-    content: '';
-    width: 0%;
-    height: 100%;
-    transition: ease 0.8s;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: none;
+  outline: none;
+  border: 0.2px solid white;
+  color: white;
+  width: 30px;
+  height: 15px;
+  padding: 20px 60px;
+  cursor: pointer;
+  font-size: 21px;
+}
+
+.selection-rock,
+.selection-paper,
+.selection-scissors {
+  position: relative;
+  overflow: hidden;
+}
```

```suggestion
.selection-weapon {
  position: relative;
  overflow: hidden;
}
```
et après tes class tu fais 
`<div class="selection-weapon selection-rock" />` pour évité d'être redandant

---

Bonne idée ca